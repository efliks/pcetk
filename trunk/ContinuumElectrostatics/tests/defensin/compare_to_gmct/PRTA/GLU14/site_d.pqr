ATOM      0  CG  GLU    14      31.638  34.411   7.434  -0.28000   1.700      0      0      0      0
ATOM      0  HG1 GLU    14      31.526  34.938   8.408   0.09000   1.000      0      0      0      0
ATOM      0  HG2 GLU    14      32.459  34.915   6.875   0.09000   1.000      0      0      0      0
ATOM      0  CD  GLU    14      30.387  34.622   6.658   0.62000   1.700      0      0      0      0
ATOM      0  OE1 GLU    14      29.951  33.671   6.003  -0.76000   1.500      0      0      0      0
ATOM      0  OE2 GLU    14      29.838  35.691   6.694  -0.76000   1.500      0      0      0      0
