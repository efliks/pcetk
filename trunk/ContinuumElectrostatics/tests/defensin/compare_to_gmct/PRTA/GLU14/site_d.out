Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named scratch/PRTA/GLU14/site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (24.1776, 30.1496, 6.10909)) 121 2
Finer level:    (30.9665, 34.7081, 7.01209) 101 1
Finer level:    (30.9665, 34.7081, 7.01209) 101 0.5
Finest level:   (30.9665, 34.7081, 7.01209) 101 0.25


Self energy of scratch/PRTA/GLU14/site_d in three dielectric environment = 419.188
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/GLU14/site_d with background charge set in three dielectric environment = -3.11491
(probably in kcal/mole)



Interaction energies of scratch/PRTA/GLU14/site_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.030675
0   1 : 0.165
1   0 : 0.0288457
1   1 : 0.689333
2   0 : 0.0538577
2   1 : 1.07181
3   0 : 0.0403949
3   1 : 0.385336
4   0 : -13.5054
4   1 : -2.65525
5   0 : -0.257023
5   1 : 4.68063
6   0 : 483.707
6   1 : 838.386
7   0 : -0.116771
7   1 : -0.00346485
8   0 : -0.188403
8   1 : -0.00689358
9   0 : -0.0295114
9   1 : 0.263308
10   0 : -0.0130634
10   1 : 0.342991
11   0 : 0.00660778
11   1 : 0.0784825
12   0 : -0.0492028
12   1 : -6.38944e-05
13   0 : -1.29822
13   1 : 7.90726
14   0 : 0.156012
14   1 : 0.926578
Total runtime: 9.71 s user 0.06 s system 
